tests/test_integration_all_rust.py

```diff
@@ -957,7 +957,6 @@ def test_empty_message_with_crypto_headers(self):
 
         yield self.shut_down(client)
 
-    """
```

Comment:
Can this test be re-enabled now that we reverted the 202/201 change?

Probably better as a separate issue

Reply:
Honestly, it's another weird one. Locally, this passes fine, however on Circle it fails with the first `send_notification()` returning a 202. I have no clue why. 

I'll file an